data-processor/src/main/java/io/micronaut/data/processor/visitors/MappedEntityVisitor.java

```diff
@@ -287,4 +295,25 @@ private void useJoinColumnNameIfSet(AnnotationMetadata annotationMetadata, Prope
             propertyElement.annotate(MappedProperty.class, builder -> builder.member(AnnotationMetadata.VALUE_MEMBER, joinColumnName));
         }
     }
+
+    /**
+     * An identity field for Oracle duality Json View has to be '_id' so we are verifying and configuring it here.
+     *
+     * @param identity the identity field
+     */
+    private void handleJsonViewIdentity(SourcePersistentProperty identity) {
+        PropertyElement identityPropertyElement = identity.getPropertyElement();
```

Comment:
I thought we could add annotation so `@Id` annotated field will have `_id` field and serde name since now identifier in Json View is expected to be named `_id` (since Oracle 23ai it seems).
Other option is to document it and let user annotate, but this is maybe better.

Reply:
Right, it is expected `_id` to be at the root document of the Json View: https://docs.oracle.com/en/database/oracle/oracle-database/23/jsnvu/introduction-car-racing-duality-views-example.html#GUID-8F192730-7885-40CF-96BE-7D6DC094A838. I meant we could document users to annotate it with `_id` but this is better approach now probably.